src/components/orderDetails/OrderDetails.tsx

```diff
@@ -0,0 +1,299 @@
+  const [itemTableData, setItemTableData] = useState<CommerceItemData[]>([]);
+
+  const { orderId } = useParams<{ orderId: string }>();
+
+  useEffect(() => {
+    setIsLoading(true);
+    getOrderData();
+    getOmsOrderStatus();
+  }, [orderId]);
+
+  const getOrderData = () => {
+    // const data: OrderData = await fetchData(
+    //   `${URL_ORDER_DETAILS}/${orderId}`,
+    //   {},
+    // );
+    console.log(orderId);
+    const data: OrderData = orderId === "60577546279" ? orderDataJSON : {};
+
+    setOrderData(data || {});
+    if (data?.commerceItem?.length) {
+      const convertedArray: CommerceItemData[] = data.commerceItem.map(
+        (item: CommerceItem) => {
+          return {
+            SKU: item.catalogRefId,
+            Type: item.itemClassType,
+            Quantity: item.quantity,
+            "Unit Price": item.priceInfo.listPrice,
+            "Total Price": item.priceInfo.rawTotalPrice,
+          };
+        },
+      );
+      setItemTableData(convertedArray);
+    }
+    setIsLoading(false);
+  };
+
+  const getOmsOrderStatus = async () => {
+    // const data: OmsOrderStatus = await fetchData(
+    //   `${URL_OMS_ORDER_STATUS}/${orderId}`,
+    // );
+    const data: OmsOrderStatus =
+      orderId === "60577546279" ? omsOrderStatusJSON : {};
+    setOmsOrderStatus(data || {});
+  };
+
+  const showPromotions = (): void => {
+    //show promotion modal
+  };
+
+  const showOmsStatusInfo = (): void => {
+    //show oms modal
+  };
+
+  const showOrderTimeline = (): void => {
+    //show order timeline modal
+  };
+
+  return isLoading ? (
+    <Loader />
+  ) : Object.keys(orderData).length > 0 ? (
+    <>
+      <div className="grid gap-y-0 grid-cols-1 m-4 p-0 sm:p-4 bg-[#ffffff] sm:bg-[#f4f4f4] rounded-md">
+        <div className="flex justify-between">
+          <span className="text-base text-slate-600 font-bold">
+            {ORDER_DETAILS}
+          </span>
+          <span
+            className="flex items-center !text-[12px] text-slate-500 font-normal cursor-pointer"
+            onClick={showPromotions}
+          >
+            <CustomIcon
+              className="mr-1"
+              alt="promotion-icon"
+              src={PromotionsIcon}
+              width="12px"
+              height="12px"
+            />
+            {PROMOTIONS}
+          </span>
+        </div>
+        <div className="flex flex-col border-t-0 sm:border-t border-solid border-[#D4D4D4] sm:grid sm:gap-y-0 sm:grid-cols-2 ">
+          <div className="flex justify-between w-full py-1 px-3 sm:p-0 border-b border-solid border-[#D4D4D4] sm:border-none justify-start bg-[#f4f4f4] rounded-t-md">
+            <span className="w-auto sm:w-1/5 flex items-center">
+              {ORDER}
+              <CustomIcon
+                className="ml-2 cursor-pointer"
+                alt="order-clock-icon"
+                src={OrderClockIcon}
+                width="12px"
```

Comment:
keep in `vw` and `vh` to maintain responsiveness.

Reply:
Converted to corresponding rem values which is .75rem